examples/elasticsearch/README.md

```diff
@@ -33,206 +33,130 @@ Documentation for other releases can be found at
-apiVersion: v1
-kind: ReplicationController
-metadata:
-  labels:
-    name: music-db
-    namespace: mytunes
-  name: music-db
-spec:
-  replicas: 4
-  selector:
-    name: music-db
-  template:
-    metadata:
-      labels:
-         name: music-db
-    spec:
-      containers:
-      - name: es
-        image: kubernetes/elasticsearch:1.2
-        env:
-          - name: "CLUSTER_NAME"
-            value: "mytunes-db"
-          - name: "SELECTOR"
-            value: "name=music-db"
-          - name: "NAMESPACE"
-            value: "mytunes"
-        ports:
-        - name: es
-          containerPort: 9200
-        - name: es-transport
-          containerPort: 9300
-```
+Kubernetes makes it trivial for anyone to easily build and scale [Elasticsearch](http://www.elasticsearch.org/) clusters. Here, you'll find how to do so.
+Current Elasticsearch version is `1.7.1`.
 
-[Download example](music-rc.yaml)
-<!-- END MUNGE: EXAMPLE music-rc.yaml -->
-
-The `CLUSTER_NAME` variable gives a name to the cluster and allows multiple separate clusters to
-exist in the same namespace.
-The `SELECTOR` variable should be set to a label query that identifies the Elasticsearch
-nodes that should participate in this cluster. For our example we specify `name=music-db` to
-match all pods that have the label `name` set to the value `music-db`.
-The `NAMESPACE` variable identifies the namespace
-to be used to search for Elasticsearch pods and this should be the same as the namespace specified
-for the replication controller (in this case `mytunes`).
-
-
-Replace `NAMESPACE` with the actual namespace to be used. In this example we shall use
-the namespace `mytunes`.
-
-```yaml
-kind: Namespace
-apiVersion: v1
-metadata:
-  name: mytunes
-  labels:
-    name: mytunes
-```
+[A more robust example that follows Elasticsearch best-practices of separating nodes concern is also available](./production_cluster/README.md).
 
-First, let's create the namespace:
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING" width="25" height="25"> Current pod descriptors use an `emptyDir` for storing data in each data node container. This is meant to be for the sake of simplicity and [should be adapted according to your storage needs](../../docs/design/persistent-storage.md).
 
-```console
-$ kubectl create -f examples/elasticsearch/mytunes-namespace.yaml 
-namespaces/mytunes
-```
+## Docker image
+
+This example uses [this pre-built image](https://github.com/pires/docker-elasticsearch-kubernetes) will not be supported. Feel free to fork to fit your own needs, but mind yourself that you will need to change Kubernetes descriptors accordingly.
+
+## Deploy
 
-Now you are ready to create the replication controller which will then create the pods:
+Let's kickstart our cluster with 1 instance of Elasticsearch.
 
-```console
-$ kubectl create -f examples/elasticsearch/music-rc.yaml --namespace=mytunes
-replicationcontrollers/music-db
+```
+kubectl create -f service-account.yaml
```

Comment:
The reason is so that docs are validated against files passed in to `kubectl ... -f ...` so commands in docs actually work for users. The previous example used a path relative to examples so this is at least better than that IMO.